templates/managed/index.html

```diff
@@ -1,450 +1,587 @@
-                        ) | safe
-                    }}
-                    <h3 class="p-heading--four"><a href="/kubernetes/managed">Kubernetes</a></h3>
-                    <p>Deploy Kubernetes on bare metal, public cloud or on top of your OpenStack, and maintain your focus on developing your cloud native applications.</p>
-                    <p><a href="/kubernetes/managed">Managed Kubernetes&nbsp;&rsaquo;</a></p>
-                </div>
-
-                <div class="col-4 u-sv3">
-                    {{
-                        image(
-                        url="https://assets.ubuntu.com/v1/317680dc-CLOUD+APPS.svg",
-                        alt="Kubernetes cloud",
-                        width="278",
-                        height="150",
-                        hi_def=True,
-                        loading="lazy",
-                        attrs={"class":"u-hide--small"},
-                        ) | safe
-                    }}
-                    <h3 class="p-heading--four"><a href="/managed/apps">Applications</a></h3>
-                    <p>Focus on your application layer and offload the dependencies with our managed platform services that provide predictable pricing from the industry leaders in Open Source.</p>
-                    <p><a href="/managed/apps">Managed Applications&nbsp;&rsaquo;</a></p>
-                </div>
-            </div>
-        </div>
+
+<div class="u-fixed-width">
+  <hr>
+</div>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h3>Experience & Consultation</h3>
+      <p>Use best practices and expert-level business consultancy to achieve your goals as fast as possible. Better economics with the rapid time to market.</p>
+      <p>
+        <a href="/engage/managed-services-overcoming-cio-challenges-2021">Read the "Managed IT Services" operations guide</a>
+      </p>
     </div>
+    <div class="col-4 u-align--center u-hide--small">
+      {{ 
+        image (
+        url="https://assets.ubuntu.com/v1/b0af9ede-UA_24-7_Support.svg",
+        alt="",
+        width="162",
+        height="162",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
     </div>
+  </div>
 </section>
-<hr>
+
 <section class="p-strip">
-    <div class="row">
-        <div class="col-6">
-            <h2>
-                Canonical as your managed service provider
-            </h2>
-        </div>
-        <div>
-            <p>
-                Canonical is not a typical managed service provider (MSP). At Canonical, the teams managing your cloud are working side by side everyday with the teams developing, maintaining and supporting the open source software you use. This guarantees resolution of any issues or bug fixes associated with your cloud service in the shortest possible time.
-            </p>
-        </div>
+  <div class="row u-sv3">
+    <div class="col-8">
+      <h2>Operations can be smarter</h2>
```

Comment:
I think this section needs more separation from the one above as they sort of merge into one.